tower_cli/resources/job.py

```diff
@@ -74,8 +74,12 @@ class Resource(models.ExeResource):
     @click.option('--verbosity', type=int, required=False, help='Specify verbosity of the playbook run.')
     @click.option('--inventory', required=False, type=types.Related('inventory'),
                   help='Specify inventory for job template to run.')
-    @click.option('--credential', required=False, type=types.Related('credential'),
-                  help='Specify machine credential for job template to run.')
+    @click.option('--credential', required=False, multiple=True, type=types.Related('credential'),
+                  help='Specify machine credential(s) for job template to run.')
+    @click.option('--vault-credential', required=False, type=types.Related('credential'),
+                  help='Specify vault credential for job template to run.')
+    @click.option('--cloud-credential', required=False, type=types.Related('credential'),
```

Comment:
I'm trying to decide if I want to provide this, or just go all-in on `--credentials`.

Reply:
I agree, we should axe cloud & vault here.

However, the `--credential` flag in this PR does what you want it to do, the plural just isn't how click handles multiples. The pattern is `--credential="cred1" --credential="cred2"`.